src/Fixer/Import/SingleImportPerStatementFixer.php

```diff
@@ -78,14 +78,13 @@ public function isCandidate(Tokens $tokens): bool
     protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
     {
         $tokensAnalyzer = new TokensAnalyzer($tokens);
-        $fixGroups = $this->configuration['group_to_single_imports'];
 
         foreach (array_reverse($tokensAnalyzer->getImportUseIndexes()) as $index) {
             $endIndex = $tokens->getNextTokenOfKind($index, [';', [T_CLOSE_TAG]]);
             $groupClose = $tokens->getPrevMeaningfulToken($endIndex);
 
             if ($tokens[$groupClose]->isGivenKind(CT::T_GROUP_IMPORT_BRACE_CLOSE)) {
-                if ($fixGroups) {
+                if ($this->configuration['group_to_single_imports']) {
```

Comment:
Did moving it here changed PHPStan issues? If so, isn't it a bug?

Reply:
It did not fix anything, it was just unnecessary assignment which I removed. Initially I added explicit bool comparison there, but in the end I removed it, but kept refactoring.